### ISSUE TITLE
Brittle stone not smeltable

### ISSUE THREAD

**Issue by Aalandriel** [CyanideX/InfernalSkies#27]
Not sure if bug or not. But brittle stone cannot be smelted to get lava. If not a bug you have to be really lucky to get 15 cobblestone (5 for barrel + 10 for lava) and go pick up the water on the second island, and NOT create stone compost until you have a cobble gen. (Quest sequence could be better then)

**Comment by CyanideX** [comment]
Yeah, I assumed people would opt for the cobble gen first but I shouldn't have assumed... I will be making the brittle stone smeltable among a few other tweaks.

**Comment by Aalandriel** [comment]
Personally I like having to go to the other island/ wait for rain as I described above. But it wasn't clear from the quest progression that, that was the order you where going for seeing as the sifting quest mentions creating the composter and hints at the other island being too dangerous e.g. not needed for progression.

**Comment by valsnoisytoys** [comment]
FWIW, without brittle stone being meltable, I would have been hooped (since it couldn't yet be melted, I exchanged 10 for a bucket of lava in creative so I could move forward). After making my stone barrel, I had only 7 cobblestone - not enough to make 1 bucket of lava.  At that point I hadn't made anything else out of cobble - I just had a combination of horrible luck sifting and a creeper blowing up 2 or 3 of my loam before I knew I needed to collect it for something.

**Comment by CyanideX** [comment]
You should get roughly 16 cobblestone from the loamy dirt... but I will double check. I made it through a few play-throughs. Maybe you lost some of the dirt?

**Comment by valsnoisytoys** [comment]
/me points to the creeper blast part of her comment. ;)

**Comment by Tzian** [comment]
I don't seem to be able to figure out how to get the cobble to smelt, i have a torch underneath it but can only get 2 of anything to go into the barrel, have waited ages to see if it would part melt and could then add another piece but to no avail
